src/Routing/RecurlyRouteSubscriber.php

```diff
@@ -119,12 +118,11 @@ protected function alterRoutes(RouteCollection $collection) {
           array(
             '_controller' => '\Drupal\recurly\Controller\RecurlySubscriptionSelectPlanController::planSelect',
             '_title' => \Drupal::config('recurly.settings')->get('recurly_subscription_max') == 1 ? 'Signup' : 'Add plan',
-            'operation' => 'select_plan',
           ),
           array(
             '_entity_access' => "$entity_type_id.update",
             '_access_check_recurly_user' => 'TRUE',
-            '_access_check_recurly_default' => 'TRUE',
```

Comment:
@markdorison No. The default just checks if there is a local account. The other access checks are checking for the local account and something else. The default is just a basic check. I actually think this PR is ready to go after watching how the access checks work in the d7 branch compared to this one.